README.md

```diff
@@ -114,14 +114,10 @@ Set `TF_VAR_gcp_project` to something unique, perhaps by appending `-2` to your
 
 #### DNS/TLS Setup
 
-Once everything is applied, run
-
-```shell
-kubectl get service --namespace ingress-controller deployment-endpoint
-```
-
-the "EXTERNAL-IP" field is the IP of your deployment.
+Once everything is applied, you should see an output named `cluster_ip`.
+Due to the "convergent" nature of Kubernetes, it may take a few minutes before this output appears.
 It's up to you to configure the DNS for your root URL domain name to point to this IP.
+For a `taskcluster-dev.net` subdomain, just add an A record to the Route53 domain in the team's staging AWS account.
```

Comment:
Could/should we do this with terraform directly eventually? Happy for that to be a bug rather than hold this up.

Reply:
Yes, as long as it's optional.  So something like taskcluster_dev_net_name = "dustin", and if that's not set it doesn't try to edit anything.  File a bug?